TACC/Core-Styles
# Fix/revert-pr-377-manually-since-design-tokens

## Overview

Revert #377 i.e. restore old v1 Portal message styles.

## Related

- expected by https://github.com/TACC/tup-ui/pull/476

## Changes

- **added** v1 message mixins and styles

## Testing

1. Open http://localhost:3000/components/detail/c-message--scope-portal.
2. Verify messages are v1 style.
3. Open http://localhost:3000/components/detail/c-form--portal.
4. Click [Toggle Field Errors] button.
5. Review form error (scroll to top) and field errors are v1 style.
3. Open http://localhost:3000/trumps/detail/s-form--portal.
4. Click [Toggle Field Errors] button.
5. Review form error (scroll to top) and field errors are v1 style.

## UI

| - | original v1 | restored v1 |
| - | - | - |
| message | ![portal message - before](https://github.com/user-attachments/assets/1e26a8d1-b006-48aa-9977-cd26f1b9a2f1) | ![portal message - after](https://github.com/user-attachments/assets/7ac5ff64-fc2f-49a3-a654-f81455e7bd93) |
| c-form | <img width="920" alt="c-form message - before" src="https://github.com/user-attachments/assets/5995d6f4-2d3f-4307-9c38-9c6a578648e2"> | <img width="920" alt="c-form message - after" src="https://github.com/user-attachments/assets/1a43cbb4-57e4-4027-8470-3c7c1907d6d9"> |
| s-form | <img width="920" alt="s-form message - before" src="https://github.com/user-attachments/assets/d0d2d567-cde9-468f-bbda-7ec87f24db53"> | <img width="920" alt="s-form message - after" src="https://github.com/user-attachments/assets/740a0dcc-77e4-44ba-9e23-426e7933f190"> |
